package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "fp-ts-training",
+  "name": "Effect training",
```

Comment:
it's still an fp-ts training repo no? You added the effect dependencies but they are not used at all, right?
I think it would be great to split the work in different pull requests and only move to pnpm in this one